package ru.karpyzin.data.error

class CepkaException {
    companion object {
        const val ERROR_WHILE_REQUEST = "Error while request. Try again later."
    }
}